### PR TITLE
Task-54: Migrate-Change-Password

### PR DESCRIPTION
**Pull request by sergevei**

### CODE REVIEW

**Review by OlegPanasyuk — APPROVED** [review]

File: src/containers/profile/accountTab/index.jsx [hunk 1]
```diff
@@ -1,47 +1,140 @@
...
+      },
+    } = this.props;
+    const { password, confirmPassword } = this.state;
+
+    const errors = changePasswordValidate(password, confirmPassword);
+
+    if (!isEmpty(errors)) {
+      this.setState({ errors });
+    } else {
+      this.setState({ update: !this.state.update });
```
Comment (universezzz) [inline src/containers/profile/accountTab/index.jsx]: pls, combine two setStates in one

File: src/containers/profile/accountTab/index.jsx [hunk 2]
```diff
@@ -1,47 +1,140 @@
...
-              <>
-                <button type='submit'  onClick={updatePassword}>save</button>
-                <button type='button' onClick={updatePassword}>close</button>
-              </>
-            )}
+  render() {
+    const { password, confirmPassword, errors, update } = this.state;
+    const {
+      account,
+      pass,
```
Comment (universezzz) [inline src/containers/profile/accountTab/index.jsx]: pls rename this field. I think pass isn't correct class name.

Comment (sergevei) [reply src/containers/profile/accountTab/index.jsx]: done

File: src/containers/profile/accountTab/index.jsx (hunk 1 above)
Comment (sergevei) [reply src/containers/profile/accountTab/index.jsx]: done

File: src/containers/profile/accountTab/index.jsx [hunk 3]
```diff
@@ -1,47 +1,138 @@
...
+    confirmPassword: '',
+    update: true,
+    errors: {},
+  };
+
+  onSubmit = event => {
+    event.preventDefault();
+    const {
+      user: {
+        user: { id },
```
Comment (universezzz) [inline src/containers/profile/accountTab/index.jsx]: rewrite to data: { id }

File: src/containers/profile/accountTab/index.jsx [hunk 4]
```diff
@@ -1,47 +1,138 @@
...
+                placeholder="Confirm Password"
+                disabled={update}
+                onChange={this.onChange}
+              />
+            </div>
+            <div className={buttons_block}>
+              {update ? (
+                <div>
+                  <button type="button" onClick={this.updatePassword}>
+                    update password
```
Comment (universezzz) [inline src/containers/profile/accountTab/index.jsx]: update password -> Update password

**Review by universezzz — APPROVED** [review]

**Pull request merged**